src/corppa/poetry_detection/annotation/recipe.py

```diff
@@ -38,6 +39,9 @@
         "hide_true_newline_tokens": False,
     },
     "global_css_dir": CURRENT_DIR,
+    # Task annotation flags, these should be move elsewhere
+    "feed_overlap": False,
+    "annotations_per_tasks": 2,
```

Comment:
@laurejt should this be `annotations_per_task` rather than task**s** ?

Reply:
Yes

Reply:
Sorry I didn't catch that yesterday! Reviewed too quickly, I guess